### PR TITLE
Add Vue integration

### LINKED ISSUE #6: Vue.js Integration

### PR DESCRIPTION
**Pull request by xadamy**
Closes #6.

### CODE REVIEW

File: packages/vue/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      $options: { propsData }
+    } = vm
+
+    const span = appsignal.createSpan()
+
+    span
+      .setAction(tag || DEFAULT_ACTION)
+      .setTags({ framework: "Vue", info, version })
+      .setParams(propsData || {})
```
Comment (matsimitsu) [inline packages/vue/src/index.ts]: Just realised something, in the API spec we treat params as a hash(map) of <string,string> does this work with the propsData here (e..g can we guarantee that a propsData object doesn't have values that are also objects?)

Comment (xadamy) [reply packages/vue/src/index.ts]: Ah yeah, you're right, I've removed this option

Comment (matsimitsu) [reply packages/vue/src/index.ts]: ok! Maybe we can create a helper at some point that ensures it's a key/value string/string hashmap :)

**Comment by xadamy** [comment]
Opened https://github.com/appsignal/appsignal-examples/issues/4 to create an example app. 🎉

**Pull request merged**